docs/DEPLOY.md

```diff
@@ -0,0 +1,3 @@
+# Deployment Instructions
+
+see <https://ops-guide.crates.io/>
```

Comment:
I'm not sure this file is really needed.

Reply:
I don't know. I think the link to that ops guide should be *somewhere* in the repo, but I'm open to alternative locations.

Reply:
It seems that this document requires authorization to access? Maybe we need to specify that only team members can access it?

Reply:
@hi-rustin yep, that's why I put "private" in the PR title 😅

Reply:
Oh, sorry! I didn't notice the `private`.

Reply:
removed the file